core/src/main/java/tachyon/UserInfo.java

```diff
@@ -25,24 +25,14 @@
 public class UserInfo {
   private final long mUserId;
 
-  private long mOwnBytes;
   private long mLastHeartbeatMs;
 
   public UserInfo(long userId) {
     Preconditions.checkArgument(userId > 0, "Invalid user id " + userId);
     mUserId = userId;
-    mOwnBytes = 0;
```

Comment:
when the user leaves, checkStatus will call cleanUserResources in StorageDir automatically, clean all resources owned by the user.